docs/dev/release.md

```diff
@@ -39,31 +42,51 @@ All bundles/package manifests for Operators for operatorhub.io reside in:
 
 #### 1. OperatorHub
 
-Fork the following repo into your own:
-  https://github.com/k8s-operatorhub/community-operators
```

Comment:
This is not mandatory, to be honest. You can create a PR from your own fork of this repo